nessie/src/test/java/org/apache/iceberg/nessie/TestNessieIcebergClient.java

```diff
@@ -151,7 +151,7 @@ public void testFindImpliedNamespacesDisabled() {
     String branch = "testFindImpliedNamespacesDisabled";
     NessieIcebergClient client =
         new NessieIcebergClient(
-            api, branch, null, ImmutableMap.of(NessieUtil.CREATE_IMPLIED_NAMESPACES, "false"));
+            api, branch, null, ImmutableMap.of(NessieUtil.IGNORE_IMPLIED_NAMESPACES, "true"));
```

Comment:
I thought only we rename the property. Keep ignore as `false` by default,

Reply:
well, I guess it makes sense to assume unset as `false`... it's intuitive for end users, I guess. Was it not your initial idea?

Reply:
Maybe I didn't see clearly. 

The latest change is ok for me.